runtime/devnet/src/config/contracts.rs

```diff
@@ -16,6 +16,7 @@ fn schedule<T: pallet_contracts::Config>() -> pallet_contracts::Schedule<T> {
 	pallet_contracts::Schedule {
 		limits: pallet_contracts::Limits {
 			runtime_memory: 1024 * 1024 * 1024,
+			validator_runtime_memory: 2 * 1024 * 1024 * 1024,
```

Comment:
The integrity test was failing:
```
failures:

---- __construct_runtime_integrity_test::runtime_integrity_tests stdout ----
thread '__construct_runtime_integrity_test::runtime_integrity_tests' panicked at /Users/R0GUE/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pallet-contracts-38.0.0/src/lib.rs:707:13:
Maximal storage size 143942352 exceeds the storage limit 0
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

@peterwht do you remember why you put this in the runtimes. The SDK uses the [default](https://github.com/paritytech/polkadot-sdk/blob/743dc632fd6115b408376a6e4efe815bd804cd52/substrate/bin/node/runtime/src/lib.rs#L1344)

Reply:
It had to be bigger than the other value so not much thought put towards it. 

I changed the contracts configuration back to default. As we might transition to only using revive I suggest we look into this when we add revive to the runtime and take a closer look at those configurations.

Reply:
https://github.com/r0gue-io/pop-node/issues/415

Reply:
I changed it back to what we had and multiply the `runtime_validator_memory` with 2, also been done in the substrate contracts node: https://github.com/paritytech/substrate-contracts-node/blob/f209befc88cb54ff50b7483c13d19e62213d0c60/runtime/src/contracts_config.rs#L31

My understanding is that `runtime_memory` is the maximum memory available during a contract call in the runtime, while `validator_runtime_memory` needs to be larger because validators must handle additional overhead like PoV data, events, and other block-related resources.